README.md

```diff
@@ -5,40 +5,25 @@
 
 # SMTP Integrator Operator
 
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
-deploying and managing a SMTP Integrator on Kubernetes and bare metal. SMTP
-is an Internet standard communication protocol for email transmission.
-
-This charm simplifies configuration of SMTP client by providing a single point
-of configuration for all the requirers using the same SMTP server. It can be
-deployed on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io)
-to [Charmed Kubernetes](https://ubuntu.com/kubernetes), public cloud Kubernetes
-offerings and virtual machines or bare metal.
-
-As such, the charm makes it easy to manage and propagate SMTP configuration, while
-giving the freedom to deploy on the substrate of their choice.
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) 
+for deploying and managing an SMTP integrator on Kubernetes or bare metal. 
+The SMTP integrator provides SMTP connection information to other charms 
+that require `smtp` integration, facilitating the email-sending capabilities 
+of those charms.
 
 For DevOps or SRE teams this charm will make operating any charm requiring SMTP
 authentication simple and straightforward through Juju's clean interface.
 
-## Project and community
-
-The SMTP Integrator Operator is a member of the Ubuntu family. It's an open source
-project that warmly welcomes community projects, contributions, suggestions,
-fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
-* [Contribute](https://charmhub.io/smtp-integrator/docs/how-to-contribute)
-Thinking about using the SMTP Integrator Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
+## Get started
 
-## Contributing to this documentation
+You can follow the tutorial [here](https://charmhub.io/smtp-integrator/docs/tutorial-getting-started).
 
```

Comment:
I'm thinking about a `## Integrations` section. Are there any required integrations that the charm needs to deploy? I think they would be worth mentioning here. Otherwise, I would add a link to the integrations tab in Charmhub like

```
## Integrations

This charm can be integrated with other Juju charms and services. You can find the full 
list of integrations in [the Charmhub documentation](https://charmhub.io/smtp-integrator/integrations).
```